plugins/org.eclipse.fordiac.ide.model/src/org/eclipse/fordiac/ide/model/dataimport/ConnectionHelper.java

```diff
@@ -79,21 +111,67 @@ public void validate() {
 				connectionState.remove(ConnectionState.VALID);
 			}
 
-			if (destination != null && destInterfaceList != null) {
+			if (destinationString != null && destInterfaceList != null) {
```

Comment:
Why is this condition different to the block for the source above?

Reply:
I guess that is a copy paste error from the original code. The code here is only migrated from the original FBNetwork importer will fix that.